### PR TITLE
Push Notification Rest API

### PR DESCRIPTION
**Pull request by SalShai**

### CODE REVIEW

File: one_fm/api/api.py [hunk 1]
```diff
@@ -20,12 +22,11 @@
 def _one_fm():
     print(frappe.local.lang)
 
-
 def set_posts_active():
     posts = frappe.get_all("Operations Post", {"site": "Head Office"})
     print(posts)
     for post in posts:
-        for date in	pd.date_range(start="2021-03-01", end="2021-03-31"):
+        for date in pd.date_range(start="2021-03-01", end="2021-03-31"):
```
Comment (Ehtesham599) [inline one_fm/api/api.py]: Why do we have hard coded dates here?

File: one_fm/api/api.py [hunk 2]
```diff
@@ -150,32 +127,75 @@ def push_notification(employee_id, title, body):
...
     # See the BatchResponse reference documentation
-    # for the contents of response.
\ No newline at end of file
+    # for the contents of response.
+
+# This function is used to send notification through Firebase CLoud Message. 
+# It is a rest API that sends request to "https://fcm.googleapis.com/fcm/send"
+# Params: employee_id e.g. HR_EMP_00001
+@frappe.whitelist()
+def push_notification_rest_api(employee_id):
```
Comment (Ehtesham599) [inline one_fm/api/api.py]: This function is sending a push notification message to FCM for a checkin reminder. Please name the function and provide its description appropriately.

**Review by Ehtesham599 — CHANGES_REQUESTED** [review]
@SalShai Check inline comments.

**Review by Ehtesham599 — APPROVED** [review]
LGTM.

**Comment by Ehtesham599** [comment]
@SalShai Resolve conflicts.

**Comment by pjamsheer** [comment]
@SalShai Please rebase with master-13 and fix the conflict

**Comment by SalShai** [comment]
Conflict Resolved.👍🏼

**Pull request merged**